.vscode/settings.json

```diff
@@ -0,0 +1,7 @@
+{
+  "search.exclude": {
+    "**/node_modules": true,
+    "**/es": true,
+    "**/lib": true
+  }
+}
```

Comment:
Since `.vscode` is ignored, would it make sense to add `!.vscode/settings.json` to the `.gitignore`, so that changes to this file will get picked up?

Reply:
I generally prefer to leave it gitignore and then if you _need_ to change it, do a `git add .vscode/settings.json -f` (with the `-f` force flag) to explicitly add it. Otherwise, it's too easy for contributors' personal, local settings to get picked up in the git workflow (which I think we want to avoid).